pkg/apiserver/registry/core/shoot/strategy.go

```diff
@@ -49,18 +49,34 @@ func (shootStrategy) NamespaceScoped() bool {
 	return true
 }
 
-func (shootStrategy) PrepareForCreate(_ context.Context, obj runtime.Object) {
-	shoot := obj.(*core.Shoot)
+func (s shootStrategy) PrepareForCreate(_ context.Context, obj runtime.Object) {
```

Comment:
Not sure why we need this change, but if, then we should make this change consistent across strategy.go files and functions.